app/javascript/packs/addressSelectors.js

```diff
@@ -0,0 +1,107 @@
+class SelectAddress {
```

Comment:
`class` no funciona en viejos navegadores. Mientras no estemos planeando darles soporte, no hay problema.

Reply:
De acuerdo con mis compañeros, al utilizar babel eso no es un problema por su forma de compilar.

Reply:
Hi, thanks for the feedback, very appreciated. We weren't sure, but I think like my teammate said. If we use babel it does not matter if we use the es6 syntax, because it will compile to es5 or another desired ECMAScript version. But we would have a code that is easier to read.

Reply:
Marking as resolve. Feel free to unresolved this if you disagree. !